solarnet/peering.sh

```diff
@@ -1,9 +1,17 @@
 #!/usr/bin/env bash
 
-# Builds a cjdroute.conf snippet for peering with all nodes in secrets.yml
-#
-# Usage:
-#   ./peering.sh
+function usage {
+  echo "Builds a cjdroute.conf snippet for peering with the specified ansible hosts, using the specified password"
+  echo
+  echo "Usage:"
+  echo "  ./peering.sh <pattern> <password-name>"
+  echo "  ./peering.sh all solarnet"
+  echo "  ./peering.sh gateway alexandria"
```

Comment:
Ah yeah, just wanted to make sure which Alexandria. (It's a popular name!) :+1:



—
Sent from Mailbox

On Mon, Sep 28, 2015 at 3:29 PM, Lars Gierth <notifications@github.com>
wrote:

>> @@ -1,9 +1,17 @@
>>  #!/usr/bin/env bash
>>  
>> -# Builds a cjdroute.conf snippet for peering with all nodes in secrets.yml
>> -#
>> -# Usage:
>> -#   ./peering.sh
>> +function usage {
>> +  echo "Builds a cjdroute.conf snippet for peering with the specified ansible hosts, using the specified password"
>> +  echo
>> +  echo "Usage:"
>> +  echo "  ./peering.sh <pattern> <password-name>"
>> +  echo "  ./peering.sh all solarnet"
>> +  echo "  ./peering.sh gateway alexandria"
> Here's the battlemesh presentation: https://www.youtube.com/watch?v=llk1Mf4hRrc
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ipfs/infrastructure/pull/94/files#r40594469